products/rhel7/profiles/stig.profile

```diff
@@ -274,7 +276,7 @@ selections:
     - sshd_disable_user_known_hosts
     - chronyd_or_ntpd_set_maxpoll
     - service_firewalld_enabled
-    - display_login_attempts
+    - display_login_attempts_stig
```

Comment:
I still believe we can extend `display_login_attempts` to accommodate any change required for STIG. Can you point out the reason why you decided to split the rules?